CMakeLists.txt

```diff
@@ -178,6 +178,15 @@ set_target_properties(fmt PROPERTIES
   VERSION ${FMT_VERSION} SOVERSION ${CPACK_PACKAGE_VERSION_MAJOR}
   DEBUG_POSTFIX d)
 
+# Configure pkg-config fmt.pc properly
+string(TOLOWER ${PROJECT_NAME} PROJECT_NAME_LOWER)
+if(CMAKE_BUILD_TYPE STREQUAL "Debug")
+  get_target_property(_FMT_LIB_POSTFIX ${PROJECT_NAME_LOWER} DEBUG_POSTFIX)
+  set(FMT_LIB_NAME ${PROJECT_NAME_LOWER}${_FMT_LIB_POSTFIX})
```

Comment:
I suggest defining `FMT_DEBUG_POSTFIX` and using it here and to set target postfix in line 177 above instead of getting it from the properties.

Reply:
Ok that sounds good.

Reply:
Like so?
```
set(FMT_DEBUG_POSTFIX d)

set_target_properties(fmt PROPERTIES
  OUTPUT_NAME fmt
  VERSION ${FMT_VERSION}
  SOVERSION ${CPACK_PACKAGE_VERSION_MAJOR}
  DEBUG_POSTFIX ${FMT_DEBUG_POSTFIX})

# Configure pkg-config fmt.pc properly
get_target_property(FMT_LIB_NAME fmt OUTPUT_NAME)
if(CMAKE_BUILD_TYPE STREQUAL "Debug")
  set(FMT_LIB_NAME ${FMT_LIB_NAME}${FMT_DEBUG_POSTFIX})
endif()
```

Reply:
Exactly. This simplifies the logic a bit.

Reply:
Pushed.